xcode/Subconscious/Shared/Components/Feed.swift

```diff
@@ -131,8 +246,8 @@ struct FeedModel: ModelProtocol {
         placeholder: "Search or create..."
     )
     /// Entry detail
-    var detail = MemoEditorDetailModel()
-    var stories: [Story] = []
+    var detail = DetailStackModel()
```

Comment:
Nit: might prefer "detailStack" for variable title, since "detail" means an individual detail view.